agents/vmware_soap/fence_vmware_soap.py

```diff
@@ -204,6 +204,7 @@ def set_power_status(conn, options):
 			if options["--action"] == "on":
 				fail(EC_WAITING_ON)
 			else:
+				print(ex)
```

Comment:
You should use the `logging.debug()`:
https://github.com/ClusterLabs/fence-agents/blob/main/doc/fa-dev-guide.md#logging-and-failing

That way it will only give you the exception info when you use -v/--verbose.